### PR TITLE
Image crop Open cv

### PR DESCRIPTION
**Pull request by ryuk156**
issue: https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93

About Pull request :
This pr consist of how we can crop images using Open cv by using this script you can make your own image cropper using different GUI library

Demo:

Before crop:
image

After crop:
image

### CODE REVIEW

**Comment by ryuk156** [comment]
@ankitdobhal  Review and merge Please

File: Image-Processing/Image_Crop_(resize)/Image_crop.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (disha2sinha) [inline Image-Processing/Image_Crop_(resize)/Image_crop.py]: Remove new line

File: Image-Processing/Image_Crop_(resize)/Image_crop.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import cv2 ##import the module
```
Comment (disha2sinha) [inline Image-Processing/Image_Crop_(resize)/Image_crop.py:1]: Insert new line

File: Image-Processing/Image_Crop_(resize)/Image_crop.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+img = cv2.imread("rose.jpg") ##read your image 
+
```
Comment (disha2sinha) [inline Image-Processing/Image_Crop_(resize)/Image_crop.py]: remove new line

File: Image-Processing/Image_Crop_(resize)/Image_crop.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+y=0  ##HEIGHT from
+x=0  ##WIDTH from
+h=300 ##HEIGHT
+w=510 ##WIDTH
```
Comment (disha2sinha) [inline Image-Processing/Image_Crop_(resize)/Image_crop.py]: Give space around =

**Review by disha2sinha — CHANGES_REQUESTED** [review]

**Comment by ryuk156** [comment]
@ankitdobhal review and merge, please

**Review by disha2sinha — APPROVED** [review]

**Comment by ryuk156** [comment]
@ankitdobhal review and merge

**Pull request merged**